src/api/v3/feedback/page_feedback.py

```diff
@@ -12,22 +12,21 @@ def page_feedback(data, region, language, comment, emotion, is_technical):
 
 
 def page_feedback_internal(data, region, language, comment, emotion, is_technical):
-    page_slug = data.get("slug")
-    if not page_slug:
+    page_slug = data.get("slug", None)
+    if page_slug is None or not page_slug:
```

Comment:
This is redundant.
In the get-function, None is the default.
And `page_slug is None` implies `not page_slug`, so it is enough to check `not page_slug`

Reply:
Will update.